packages/vitest/src/node/stdin.ts

```diff
@@ -119,12 +130,13 @@ export function registerConsoleShortcuts(ctx: Vitest) {
 
   async function inputFilePattern() {
     off()
-    const { filter = '' }: { filter: string } = await prompt([{
-      name: 'filter',
-      type: 'text',
-      message: 'Input filename pattern',
-      initial: latestFilename,
-    }])
```

Comment:
Thank you for reviewing!

I didn't know about `type: "autocomplete"`  for `prompts` package.

I will check your PR and explore it further.

Reply:
I will not use autocomplete for this pull request since you have already created a version with autocomplete.

Reply:
I prefer `autocomplete` approach because it provides navigation using arrow keys out of the box, although I like output better in this PR

Reply:
> I prefer `autocomplete` approach because it provides navigation using arrow keys out of the box, although I like output better in this PR

But it looks like you can't enter a pattern that matches multiple files or a single file like you can in Jest (if arrow keys are not used, it will use pattern in the input, but if something is chosen, it will run only selected file), so maybe home made solution is better in this case if we can support navigation

Reply:
@sheremet-va 
Thank you for reviewing, I will try to support arrow key navigation without using `autocomplete`.

Reply:
@sheremet-va @hi-ogawa 
I've implemented arrow key navigation.
I would greatly appreciate for your review!